### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by mathildemouw**
Closes https://github.com/doubleunion/doubleunion-dot-org/issues/32

- [x] Add newest board member Golden's bio
- [x] remove Tina's "Director of Multiculturalism and Inclusion" title.
- [x] remove Sally's board member bio
- [x] remove Mary's board member bio
- [x] change Dani's title to Treasurer

### CODE REVIEW

File: app/views/static_pages/about.html.haml [hunk 1]
```diff
@@ -38,10 +38,10 @@
...
-  %h4 Mary Becica, Board Member, Executive Director
+  =image_tag "golden.jpg"
+  %h4 Alexsarah "Golden" Collier, Board Member, Executive Director
 
-  %p Mary Becica chairs the space committee at Double Union and works on designing and building the physical space. During the day Mary is a Director of Product Management at AppDirect, where she meets with a lot of people and thinks about enterprise software management. At night she works on data visualization projects, thinks about cryptocurrency futurism, and reads a lot. You can find her at @mbecica.
+  %p Golden is a Learning Specialist at an SF elementary school by day. He is also a teaching member of Girl Army, an Oakland-based collective providing strengths-based sliding scale self defense courses for women, gender non-conforming/non-binary, and trans folks. Also, Golden is a member of Sula Collective, maker @ <a href="kidsizzle.etsy.com">kidsizzle.etsy.com</a>, apprentice-level beekeeper, motorcyclist, digital media activist with Womanist Trilliance and Combahee Radio, former world record holder, and metalsmith who enjoys dreamy hikes, practicing mindfulness, playing the viola/banjo, singing, the ocean, and libraries.
```
Comment (lilliealbert) [inline app/views/static_pages/about.html.haml]: @mathildemouw The link to Golden's Etsy page is trying to go to http://doubleunion.org/kidsizzle.etsy.com. What do you think about updating this to use the rails link_to helper and make it `target: '_blank'`?

Comment (mathildemouw) [inline app/views/static_pages/about.html.haml]: @lilliealbert thanks for catching that, I'll fix it tonight :)

**Comment by mathildemouw** [comment]
@lilliealbert I was having trouble getting the link to stay inside the paragraph, so I didn't use `link_to` in the end, but I the link does go to the right place now!

**Comment by lilliealbert** [comment]
Weird! Did you tried interpolating it like this?

```
%h4
  Cool paragraph start and #{ link_to "has a cool link in it!", "http://www.doubleunion.org" }
```

**Comment by mathildemouw** [comment]
@lilliealbert I realized what I was doing wrong before, I was trying to start a `= ruby section of haml and somehow blend it into the paragraph like this`. Your suggestion worked!

**Comment by lilliealbert** [comment]
Rebased manually and merged in as 57a9733 

Also, I snuck in a `target: "_blank"` for the link to Golden's etsy shop so it'll open in a new tab.

Also also I deployed it to staging & prod 🚀

**Pull request closed without merging**